Common/PayloadTrait.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SyliusConsumerBundle\Common;
+
+use Webmozart\Assert\Assert;
+
+trait PayloadTrait
```

Comment:
would use a `Payload` class instead of a `PayloadTrait` like we did in recent other projects